pkg/bb/cmd/main.go

```diff
@@ -20,6 +20,12 @@ func run() {
 }
 
 func main() {
+	arg1 := os.Args[0]
+	for s, err := os.Readlink(arg1); err == nil && s != "bb"; s, err = os.Readlink(arg1) {
```

Comment:
Instead of `s != "bb"`, could you use `os.Base(s) != "bb"`? Then it would work with symlinks from other directories.

Reply:
I don't see that as needed. Symlinks to bbin must be to "anything but" bb in /bbin.
So they symlink to something in bbin, and all things in bbin have a value of bb.

So it would not really help IIUC. I'm happy to be corrected.

Reply:
I agree with Ryan, let's make it work with other directories. Just because our build system doesn't do it, doesn't mean others don't (hint hint, bazel)

Reply:
you can tell me when you get in but I would like to see the case that you get to bb WITHOUT going via a symlink in /bbin. I could not think of one.